jekyll/_cci2/sso/user-sign-in.adoc

```diff
@@ -0,0 +1,28 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= User sign-in
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Instructions outlining how to sign-in as an SSO User
+:icons: font
+:experimental:
+
+NOTE: SAML Single Sign-On for CircleCI is currently in closed preview.
+
+== Accessing an SSO-enabled Organization
+
+[join-an-organization]
+=== Join an organization
+
+New users can join an organization via link:https://circleci.com/docs/manage-roles-and-permissions/#add-people-to-your-organization[an invite] or by visiting https://circleci.com/authentication/login?f=sso. In the future, a `Join with SSO` button will be added to the user homepage.
```

Comment:
I think the intended audience for this page is users wanting to access a SSK-enabled CCI org, so an alternative to the flows documented here: https://circleci.com/docs/invite-your-team/ ... Just checking this and then I will change the voice in this content to address the user directly and add a link to this page from the invite-your-team doc